.github/workflows/slither.yml

```diff
@@ -0,0 +1,59 @@
+name: Static-Analysis
+on:
+    push:
+        paths:
+            - 'contracts/**'
+            - '.github/workflows/slither.yml'
+            - '**.py'
+    pull_request:
+        paths:
+            - 'contracts/**'
+            - '.github/workflows/slither.yml'
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: [3.6]
+                node-version: [16.x]
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Branch name
+              run: echo running on branch ${GITHUB_REF##*/}
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - run: npm install
+            - run: npm run build
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v2
+              with:
+                  python-version: ${{ matrix.python-version }}
+            - name: Install dependencies
+              run: |
+                  sudo snap install solc
+                  python -m pip install --upgrade pip
+                  pip install slither-analyzer solc-select
+                  solc-select install 0.6.11
+                  solc-select use 0.6.11
+            - name: Summary of static ananlysis
+              run: |
+                  echo "Running slither"
+                  slither . --print human-summary
+            - name: Contract summary of static ananlysis
+              run: |
+                  echo "Running slither"
+                  slither . --print contract-summary
```

Comment:
I'm not familiar with all this stuff - but is there something more useful than just printing all kinds of information? Are there like warning thresholds to employ?

For example this step ("Summary of static analysis") in:
https://github.com/underfitted/bnomial-web3/runs/4622459514

I see a "high issues" count of 2... is that ok? Should the step maybe throw an error then?

Reply:
I changed the workflow. So, I just reported the issues to do solved